aggregator/src/main/java/com/redhat/cloud/notifications/DailyEmailAggregationJob.java

```diff
@@ -147,7 +144,7 @@ private void sendIt(List<AggregationCommand> aggregationCommands) {
             .withTimestamp(LocalDateTime.now(UTC))
             .withEvents(eventList)
             .withContext(new Context.ContextBuilder()
-                .withAdditionalProperty("single_daily_digest_enabled", aggregatorConfig.isSingleDailyDigestEnabled())
+                .withAdditionalProperty("single_daily_digest_enabled", true)
```

Comment:
Are we keeping or removing this? It is still set here, but no longer consumed in the engine.

Reply:
A 2 steps change would probably be safer. When this PR is deployed, an old aggregator could produce something a new engine wouldn't understand, unless the engine can still deal with old aggregation commands.

Reply:
Engine uses single daily on prod for more than a month, so we don't plan to use old aggregation commands anymore.